runtime/oti/cfr.h

```diff
@@ -505,6 +513,15 @@ typedef struct J9CfrAttributeUnknown {
     U_8* value;
 } J9CfrAttributeUnknown;
 
+typedef struct J9CfrAttributeRecord {
+    U_8 tag;
+    U_16 nameIndex;
+    U_32 length;
+    UDATA romAddress;
+    U_16 numberOfRecordComponents;
+    struct J9CfrRecordComponent* recordComponents;
```

Comment:
Can you reorganize this so it packs better?
```c
     U_8 tag;
     U_16 nameIndex;
     U_16 numberOfRecordComponents; 
     U_32 length;
     UDATA romAddress;
```

Reply:
This one needs to match the other J9CfrAttribute structures.